FrontEnd/src/components/landing-page/partners/Partners.jsx

```diff
@@ -8,12 +8,12 @@ const MainPartners = () => {
         <div className="partners-section__navi-icons">
           <div className="partners-section__navi-icon">
             <div className="partners-section__icon">
-              <img src="/svg/icon-left.svg" alt="" />
+              <img src="/svg/icon-left.svg" alt="Left" />
```

Comment:
The content of the alt attribute is literally what will be read by a screen reader, for example, if the user can not see. I would use more descriptive text.